lectures/urls.py

```diff
@@ -1,8 +1,8 @@
 from django.urls    import path
-from lectures.views import LectureDetailView, LectureLikeView
+from lectures.views import LectureDetailView, LectureLikeView,LecturesView 
 
 urlpatterns = [
     path('/<int:lecture_id>', LectureDetailView.as_view()),
     path('/<int:lecture_id>/like', LectureLikeView.as_view()),
+    path('/upload',LecturesView.as_view()),
```

Comment:
REST에 맞게 동사는 http method만 표시합니다. 따라서 수업을 등록하는 기능이기 때문에 아래와 같이 수정하면 좋을 것 같습니다.
즉, 지원님의 lecture list와 건우님의 lecture 등록이 같은 view안에 get과 post로 같이 존재해야 되어야 합니다.
이 부분을 두분이서 View이름과 url을 통일해 주세요.

```suggestion
    path('',LecturesView.as_view()),
```